services/placement/algo/sharded_helper.go

```diff
@@ -142,21 +142,27 @@ func (ph *placementHelper) GeneratePlacement() services.ServicePlacement {
 		}
 	}
 	return placement.NewPlacement().
-		SetInstances(ph.instances).
+		SetInstances(ph.getInstances()).
```

Comment:
Can we come up with a better name than `get..`? Perhaps `instanceList` or something along those lines?